ext/curl/interface.c

```diff
@@ -1151,16 +1168,50 @@ PHP_MINIT_FUNCTION(curl)
 
 #if LIBCURL_VERSION_NUM >= 0x073E00 /* Available since 7.62.0 */
 	REGISTER_CURL_CONSTANT(CURLOPT_DOH_URL);
+	REGISTER_CURL_CONSTANT(CURLOPT_UPKEEP_INTERVAL_MS);
```

Comment:
do we need a special treatment for this ? e.g. curl_easy_upkeep

Reply:
@devnexen You're totally right ! Sorry about that. I had this in a separate branch because I was wondering how to handle the case where libcurl is < 7.62.0. 

We have 2 solutions 
- register the `curl_upkeep()` function only if underlying libcurl is > 7.62.0 (but we would have to handle `curl_arginfo.h` manually to add some macro in it).
- always register `curl_upkeep()` , in the current case I returned false and sent a warning, but I don't like that the `curl_errono` is unchanged in this case.

What do you think ?

Reply:
Good question. I would tend to choose the first as it s hard to justify its existence without the constant being supported. But maybe @cmb69 would have an insight for this particular point.

Reply:
While not depending on a specific library version, even ext/standard conditionally defines some functions, e.g.
https://github.com/php/php-src/blob/c451a9bddc15ea86e10a1dbb3ccbd269fae2abf6/ext/standard/basic_functions.stub.php#L422-L424
I prefer that very much over the function doing nothing and returning false.

Reply:
@cmb69 @devnexen I'v made the change to make this function only available for libcurl 7.62+